url_shortener/views/handlers.py

```diff
@@ -126,26 +126,33 @@ def login_user(request: Request) -> Response:
             'status': 'error',
             'description': 'Username cannot be less than 4 characters.',
         })
-    user = logic.read_user_by_username(username_entered)
 
+    token = uuid.uuid4().hex
+    # TODO: if user is logged in check does NOT work!
     # Check if user is already logged in
-    if any(user.username == k.username for k in logic.get_tokens()):
+    print(request.is_authenticated) # Returns false
+    print(request.authorization) # Returns None as expected
+    print(request.authenticated_userid) # Returns None
+    print(token)
+    if request.is_authenticated:
```

Comment:
I don't know how can I check if user is logged in through this device. UUID doesn't provide a HWID.

Reply:
forgot to comment, if statement always returns False